### DISCUSSION TITLE
Upcoming v 1.0.0 release of homarr

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#887] [Q&A]
Hi
As some may have heard, homarr is next to release their v1, but under a an organisation on github (https://github.com/homarr-labs/homarr) and there will most likely be some changes needed, I've asked in their discord, if there is any automateable process to migrate homarr boards, as this process normally has to be done manually through import/export.

I'll report back here once I get some new.

Here a link to the respective thread on github: 
https://discord.com/invite/aCsmEV5RgA
https://discord.com/channels/972958686051962910/1276860857531502705/1318687593042415746

If anyone or even @havardthom who last updated this awesome script, may be able to help me a bit, that would be awesome!

**Comment by MickLesk** [comment]
& what is the Migration Part? ^^ 

Actually they dont have releases?

**Reply by CrazyWolf13** [reply]
@MickLesk 
I'm currently trying to get some answers from their discord, afaik boards have to be exported through the GUI, and then reimported through the GUI on the new app, in a special import screen, which maybe is accessible via api?

@MickLesk  they do have, but only on ghcr: https://github.com/homarr-labs/homarr/pkgs/container/homarr
They normally publish the release information on discord, as it's not yet the full v1 release.

```
docker pull ghcr.io/homarr-labs/homarr:v1.0.0-beta.3
```

They had a closed pre-testing, which I was part of, and now they made the testing public, in some weeks it will be the full release ov V1.

**Reply by MickLesk** [reply]
I'm happy to hand this over :D 

@havardthom can you take a look in the next time?

**Reply by CrazyWolf13** [reply]
Possibly due to the huge changes, it would make sense to just inform the users, when they run the update function that a new v1 is available, but they need to export create a new LXC via script and re-setup homarr using the import function, though I think this will reset the overall configuration.

Here is a list of the breaking changes, this will maybe be helpful, for the .envfile conversion.
https://deploy-preview-113--homarr-docs.netlify.app/blog/2024/09/23/version-1.0/#breaking-changes

Though it's probably a smart idea to wait for the full v1 release.

After speaking to one of the lead devs, they actually seem to like the approach of community-scrips and have asked to get in touch with them, so they potentially could include community-scripts as an "official" method of installation.

Would you @MickLesk  or @havardthom like to get in touch with them, as you both know the goals and core values of community-scrips better than me.
They prefer communication via discord, I've invited 

<details>
  <summary>Discord Username</summary>

  ```
  Manicraft1001
  ```
</details>
